src/cpu.c

```diff
@@ -111,13 +112,25 @@ unsigned int getAddressByOpcode(struct OPCODE * opcode, struct CPU *cpu, struct
   }
 }
 
+void setZeroAndNegativeFlags(struct CPU *cpu, unsigned char data) {
+  cpu->status.zero = data == 0;
+  cpu->status.negative = (data >> 7) & 1;
+}
+
+void pushStack(struct CPU *cpu, struct BUS *bus, unsigned char value) {
+  writeBus(0x0100 + cpu->stack_pointer, value, bus);
```

Comment:
Maybe we could use a static var to describe this magic number, wdyt?

Reply:
Totally agreed, I'll move this to a static var

Reply:
Moved this to const var, this should not be changed anyway